libmscore/instrchange.h

```diff
@@ -67,13 +69,16 @@ class InstrumentChange final : public TextBase {
       std::vector<Clef*> clefs() const;
       std::vector<StaffTypeChange*> staffTypeChanges() const;
 
-
       bool init() const                     { return _init; }
       void setInit(bool init)               { _init = init; }
 
       bool showWarning() const              { return _showWarning; }
-      void setShowWarning(bool showWarning) { _showWarning = showWarning; }
+      void setShowWarning(const bool showWarning) { _showWarning = showWarning; }
       void setNextChord(ChordRest* chord);
+      void setLines(const int lines)        { _lines.push_back(lines); }
```

Comment:
No need for `const int`. `const` refers to a value that shouldn't be changed. But, since lines is passed as an `int`, a copy of it will be passed (if it were `int&` or `int*`, that would be different). So, it makes no sense to add a `const` qualifier.